scripts/start-rpc-servers.ts

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env ts-node
+import { ChildProcess, exec, spawn } from "child_process";
+
+enum Participant {
+  Alice = "alice",
+  Irene = "irene",
+  Bob = "bob",
+}
+enum Color {
+  black = "[30m",
+  red = "[31m",
+  green = "[32m",
+  yellow = "[33m",
+  blue = "[34m",
+  magenta = "[35m",
+  cyan = "[36m",
+  white = "[37m",
+  gray = "[90m",
+}
+
+// This is a simple script that:
+// - starts up a local anvil chain
+// - starts up 3 local rpc servers for alice,bob, and irene
+// It can be run from the go-nitro folder with the following command:
+// npx ts-node scripts/start-rpc-servers.ts
+const chain = exec("anvil --chain-id 1337");
+chain.stdout?.on("data", (data) => {
+  console.log(data.toString());
+});
+chain.stderr?.on("data", (data) => {
+  console.log(data.toString());
+});
+
+setupRPCServer(Participant.Alice, Color.blue);
+setupRPCServer(Participant.Irene, Color.green);
+setupRPCServer(Participant.Bob, Color.yellow);
+
+function printWithColor(message: string, color: Color) {
+  console.log(`\x1b${color.toString()}${message}\x1b[0m`);
+}
+
+function setupRPCServer(participant: Participant, color: Color): ChildProcess {
+  let rpcServerProcess: ChildProcess;
+  switch (participant) {
+    case Participant.Alice:
+      rpcServerProcess = exec("go run . -autodeploy -usedurablestore");
+      break;
+    case Participant.Irene:
+      rpcServerProcess = exec(
```

Comment:
Will these processes all get tidied up nicely when the script exits?

Reply:
Yup, these child processes get automatically cleaned up when the node script complete. I've confirmed this by printing out the pid of a child process and checking it during and after running the RPC server.